volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -468,6 +542,27 @@ def get_max_fds(self) -> interfaces.objects.ObjectInterface:
 
 class mount(objects.StructType):
 
+    MNT_NOSUID = 0x01
+    MNT_NODEV = 0x02
+    MNT_NOEXEC = 0x04
+    MNT_NOATIME = 0x08
+    MNT_NODIRATIME = 0x10
+    MNT_RELATIME = 0x20
+    MNT_READONLY = 0x40
+    MNT_SHRINKABLE = 0x100
+    MNT_WRITE_HOLD = 0x200
+    MNT_SHARED = 0x1000
+    MNT_UNBINDABLE = 0x2000
+
+    MNT_FLAGS = {
+        MNT_NOSUID:     "nosuid",
+        MNT_NODEV:      "nodev",
+        MNT_NOEXEC:     "noexec",
+        MNT_NOATIME:    "noatime",
+        MNT_NODIRATIME: "nodiratime",
+        MNT_RELATIME:   "relatime",
```

Comment:
Sorry, we need to remove the multiple spaces here.

Reply:
That's ok, yapf would do it anyway.